zap/src/test/java/org/zaproxy/zap/model/StandardParameterParserUnitTest.java

```diff
@@ -51,11 +50,16 @@ public void defaultParser() {
         assertEquals(spp.getKeyValuePairSeparators(), "&");
         assertEquals(spp.getKeyValueSeparators(), "=");
         assertEquals(spp.getStructuralParameters().size(), 0);
-        Map<String, String> res = spp.parse("a=b&b=c&d=f");
```

Comment:
We should keep these and suppress the deprecations, to continue checking that they are working properly.